### ISSUE TITLE
ReferenceError: Can't find variable: waitsForPromise

### ISSUE THREAD

**Issue by mdvorscak** [steelbrain/jasmine-fix#2]
When using this library, as described in the readme:

```typescript
import { it } from 'jasmine-fix';
import { myAsyncFunction } from 'myAsyncLibrary';
 describe('functionx', function(){
        it('do a thing asynchronously', async function(){
            var sut = await myAsyncFunction();
            expect(sut.thing).toBe(myExpectation);
        });
});
```

**Comment by steelbrain** [comment]
Hi @mdvorscak 

This package depends on `waitsForPromise` being available globally, which is in the Atom env.

You can find the source for `waitsForPromise` at https://github.com/atom/atom/blob/89f1710cfe56b5277d48956106b9a8a0655431d9/spec/async-spec-helpers.js#L54-L62

Tho I could add a local polyfill in this package as well

**Comment by mdvorscak** [comment]
I'll leave it up to you how you want to handle it. For me, it's enough to just update the documentation in the README to reflect this.

**Comment by steelbrain** [comment]
jasmine-fix v1.1.0 is out with no need for global waitsForPromise
